ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3110,3 +3110,23 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- winter_outside_temperature_setpoint:
+    flexible_min: 255.9259259
+    flexible_max: 322.037037
+
+- summer_outside_temperature_setpoint:
+    flexible_min:  255.9259259
+    flexible_max: 322.037037
+
+- supply_air_differential_pressure_sensor:
+    flexible_min: -996.320096
+    flexible_max: 996.320096
+
+- supply_air_deadband_temperature_setpoint:
+    flexible_min: 255.9259259
+    flexible_max: 308.1481481
+
+- season_mode:
```

Comment:
I would change this to `season_command` as it has only two states, or `season_status` if this is a feedback point.